src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/GroupPriceType.php

```diff
@@ -0,0 +1,102 @@
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Pricing;
+
+use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class GroupPriceType extends TranslatorAwareType
+{
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $currencyByIdChoiceProvider;
+
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $countryByIdChoiceProvider;
+
+    /**
+     * @var FormChoiceProviderInterface
+     */
+    private $groupByIdChoiceProvider;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        FormChoiceProviderInterface $currencyByIdChoiceProvider,
+        FormChoiceProviderInterface $countryByIdChoiceProvider,
+        FormChoiceProviderInterface $groupByIdChoiceProvider
+    )
+    {
+        parent::__construct($translator, $locales);
+        $this->currencyByIdChoiceProvider = $currencyByIdChoiceProvider;
+        $this->countryByIdChoiceProvider = $countryByIdChoiceProvider;
+        $this->groupByIdChoiceProvider = $groupByIdChoiceProvider;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('product_id', HiddenType::class)
+            ->add('currency_id', ChoiceType::class, [
+                'label' => $this->trans('Currency', 'Admin.Global'),
+                'placeholder' => $this->trans('All currencies', 'Admin.Global'),
+                'choices' => $this->currencyByIdChoiceProvider->getChoices(),
+                'required' => false,
+            ])
+            ->add('country_id', ChoiceType::class, [
+                'label' => $this->trans('Country', 'Admin.Global'),
+                'placeholder' => $this->trans('All countries', 'Admin.Global'),
+                'choices' => $this->countryByIdChoiceProvider->getChoices(),
+                'required' => false,
+            ])
+            ->add('group_id', ChoiceType::class, [
+                'label' => $this->trans('Group', 'Admin.Global'),
+                'required' => false,
+                'placeholder' => $this->trans('All groups', 'Admin.Global'),
+                'choices' => $this->groupByIdChoiceProvider->getChoices(),
+            ]);
+    }
```

Comment:
I wonder if the `shop_id` shouldn't be integrated in this one as well? (it's only present when multishop is enabled)

Reply:
done